pages/profile.tsx

```diff
@@ -153,6 +159,18 @@ function Profile() {
                                                className="border focus:outline focus:outline-sky-500 rounded-lg px-2 py-1"
                                                type="text"/>
                                     </div>
+                                    <div className="flex flex-col gap-1">
+                                        <label className="relative inline-flex items-center cursor-pointer">
+                                        <input type="checkbox"
+                                               className="sr-only peer"
+                                               onChange={() => onChangeMailingList(mailerlist)}
+                                               checked={mailerlist}
+                                               />
+                                        <div
+                                            className="w-11 h-6 bg-gray-200 rounded-full peer peer-focus:ring-4 peer-focus:ring-blue-300 dark:peer-focus:ring-blue-800 dark:bg-gray-700 peer-checked:after:translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-0.5 after:left-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-5 after:w-5 after:transition-all dark:border-gray-600 peer-checked:bg-blue-600"></div>
```

Comment:
Reyan möchte den Hintergrund des Switches in der Farbe Sky-500 haben. (war aber nicht im Ticket spezifiziert)